<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CppTools::TidyChecks</class>
 <widget class="QWidget" name="CppTools::TidyChecks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QListWidget" name="checksList">
     <item>
      <property name="text">
       <string notr="true">android-*</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">boost-*</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">bugprone-*</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">cert-*</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">cppcoreguidelines-*</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">clang-analyzer-*</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">clang-diagnostic-*</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">google-*</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">hicpp-*</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">llvm-*</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">misc-*</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">modernize-*</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">mpi-*</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">objc-*</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">performance-*</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">readability-*</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
